### DISCUSSION TITLE
There is a  problem with markdown file viewing

### DISCUSSION THREAD

**Discussion by yanyaoli** [community/community#47151] [Mobile] [answered]
I wrote a markdown text file (the content is in Chinese), which is smaller than other files, but it reminds me that I cannot view the file, while other files can be viewed normally. Why?

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, 版本 1.99.0 (90728769)
Device: iPhone Xʀ (iOS 16.3)
</details>

**Comment by yanyaoli** [comment, marked as answer]
**I found a solution:**
Change your file name from Chinese to English, and you will be able to view it normally.

**Reply by FengZhiheng** [reply]
You are right. But all my markdown files in one project are Chinese filenames. So I think GitHub need to fix this bug.

**Reply by yanyaoli** [reply]
> You are right. But all my markdown files in one project are Chinese filenames. So I think GitHub need to fix this bug. 

Yes, it would be great if GitHub could fix this bug.😝

**Reply by anjerukare** [reply]
Same with Russian file names. I hope this will be fixed.

**Reply by L-Super** [reply]
> You are right. But all my markdown files in one project are Chinese filenames. So I think GitHub need to fix this bug. 

Me too

**Reply by anjerukare** [reply]
Fixed in the latest version.
Thanks, GitHub team!
